sourcecode/apis/contentauthor/app/Libraries/H5P/AjaxRequest.php

```diff
@@ -231,34 +232,44 @@ private function libraryRebuild(Request $request): array
             throw new UnauthorizedHttpException("Not logged in");
         }
 
+        /** @var H5PLibrary $H5PLibrary */
         $H5PLibrary = H5PLibrary::findOrFail($request->input('libraryId'));
         $framework = $this->core->h5pF;
 
         $libraries = collect();
-        $this->getLibraryDetails($H5PLibrary, $libraries)
-            ->each(function ($library) use ($framework) {
-                $framework->deleteLibraryDependencies($library['libraryId']);
+        $details = $this->getLibraryDetails($H5PLibrary, $libraries);
```

Comment:
Worth noting that `$libraries` will, perhaps surprisingly, be the same as `$details`, since `getLibraryDetails` returns the same object it receives. Therefore it's pointless to have them assigned to separate variables.